challenges/cmojidra_Headstorm/cmojidraFrontEnd/frontend.php

```diff
@@ -0,0 +1,182 @@
+<!-- Submitted by Charmish Mojidra (cmojidra@andrew.cmu.edu) -->
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="icon" href="headstorm-logo.png">
+<meta name="view" content="width=device-width, initial-scale=1">
+<style>
+	@import url('https://fonts.googleapis.com/css?family=Josefin+Sans');
+
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  outline: none;
+  font-family: 'Josefin Sans', sans-serif;
+}
+
+body{
```

Comment:
The final product on your styling is pretty good! For an extra touch, consider adding functionality for mobile/responsive design (not necessary for this PR challenge!)

Reply:
Thank you very much for your comments and due to time constraints and university work I just made the application for the desktop version, however, in a production environment, I would definitely consider using Bootstrap or Media Queries for making the websites responsive.